check/src/unify_type.rs

```diff
@@ -318,7 +318,11 @@ fn find_alias_<'a, U>(unifier: &mut UnifierState<'a, U>,
                 if l_id == r_id {
                     // If the aliases matched before going through an alias there is no need to
                     // return a replacement type
-                    return Ok(if did_alias { Some(l.clone()) } else { None });
+                    return Ok(if did_alias {
+                        Some(l.clone())
+                    } else {
+                        None
+                    });
```

Comment:
Yeah, I am only on 0.5 because there is a bug in Rustycode preventing me from using it https://github.com/saviorisdead/RustyCode/issues/164. Hoped I would still be fine on 0.5 but I guess not. Going to see if I can setup rustfmt through the test.sh script instead